env0/resource_environment.go

```diff
@@ -79,7 +79,6 @@ func resourceEnvironment() *schema.Resource {
 				Type:        schema.TypeBool,
 				Description: "redeploy only on path changes only",
 				Optional:    true,
-				Default:     true,
```

Comment:
Removed the default - true.
It might cause issues to existing deployments. Users may have to set it to false explicitly.

Reply:
> It might cause issues to existing deployments. Users may have to set it to false explicitly

Why is that? Wouldn't they just see it as a change on the first apply?

Reply:
@RLRabinowitz - yes, that's all. I just wanted to mention it.